tests/nat/profiler/test_prompt_optimizer.py

```
@@ -225,3 +227,72 @@ def fake_apply_suggestions(cfg, prompts):  # noqa: ANN001
+
+    full_space = {"prompt_param": SearchSpace(is_prompt=True, prompt="Base", prompt_purpose="Greet")}
+    run_cfg = _make_run_config(base_cfg)
+    base_cfg.functions = {
+        "init_fn": InitFunctionConfig(),
+        "recombine_fn": RecombineFunctionConfig(),
+    }
+    base_cfg.workflow = InitFunctionConfig()
+
+    class _EvalRun:
+
+        def __init__(self, config):  # noqa: ANN001
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `ANN001`)

Remove unused `noqa` directive

(RUF100)